Detectors/FIT/FT0/simulation/src/Digitizer.cxx

```diff
@@ -122,9 +125,11 @@ Digitizer::CFDOutput Digitizer::get_time(const std::vector<float>& times, float
     }
   }
   if (!result.particle) {
-    LOG(DEBUG) << "CFD failed to find peak ";
-    for (auto t : times) {
-      LOG(DEBUG) << t << ", dead time " << deadTime;
+    if (times.size() > 104) {
```

Comment:
Looks like debugging changes. Do you want to commit this?

Reply:
O, sorry, forget to remove

Reply:
I think this still needs to be removed. Or is there something particular about 104 times?